app/scripts/filters/resources.js

```diff
@@ -177,6 +177,19 @@ angular.module('openshiftConsole')
       return null;
     };
   })
+  .filter('isWebhookAvailable', function(SecretsService) {
```

Comment:
I'd suggest having the `webhookURL` filter return an empty string instead of creating a separate filter.